src/style.css

```diff
@@ -0,0 +1,113 @@
+* {
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  height: 403px;
+  width: 200px;
+  display: block;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  padding: 29px 19px 41px;
+}
+
+.image_photo {
+  height: 134px;
+  width: 160px;
+}
+
+.card__name {
+  padding: 0 14px;
+  margin-bottom: 6px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__code {
+  padding: 0 14px;
+  margin-bottom: 15px;
+  font-weight: 400;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__rating {
+  justify-content: space-between;
+  display: flex;
+  padding: 0 14px;
+  margin-bottom: 20px;
+}
+
+.rating__stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars > :nth-child(n + 5) {
+  background-image: url(./images/star.svg);
+}
+
+.rating__reviews {
+  font-weight: 400;
+  font-size: 11px;
```

Comment:
You are repeating `font-size: 11px;` three times in your code. Let's move it to `body`